centreon/packages/ui/src/components/DataTable/Item/DataTableItem.tsx

```diff
@@ -66,33 +86,64 @@ const DataTableItem = forwardRef(
         </ActionArea>
         {hasActions && (
           <MuiCardActions>
+            <span />
             <span>
+              {onEditAccessRights && (
+                <IconButton
+                  ariaLabel={labels.labelShareWithContacts}
+                  title={labels.labelShareWithContacts}
+                  onClick={pipe(onEditAccessRights, closeMoreActions)}
+                >
+                  <ShareIcon fontSize="small" />
+                </IconButton>
+              )}
               <IconButton
-                aria-label="delete"
-                data-testid="delete"
-                icon={<DeleteIcon />}
-                size="small"
-                variant="ghost"
-                onClick={onDelete}
-              />
-            </span>
-            <span>
-              <IconButton
-                aria-label="edit access rights"
-                data-testid="edit-access-rights"
-                icon={<ShareIcon />}
-                size="small"
-                variant="primary"
-                onClick={() => onEditAccessRights?.()}
-              />
-              <IconButton
-                aria-label="edit"
-                data-testid="edit"
-                icon={<SettingsIcon />}
-                size="small"
-                variant="primary"
-                onClick={() => onEdit?.()}
-              />
+                ariaLabel={labels.labelMoreActions}
+                title={labels.labelMoreActions}
+                onClick={openMoreActions}
+              >
+                <MoreIcon />
+              </IconButton>
+              <Menu
+                anchorEl={moreActionsOpen}
+                open={Boolean(moreActionsOpen)}
+                onClose={closeMoreActions}
+              >
+                <ActionsList
+                  actions={[
+                    ...(onEdit
```

Comment:
@bdauria was right. IMO, the best option is to make the more action generic. Why not defining a prop that allow the develop to a list of icons, labels and actions?